LanguageFeatures/Inline-classes/syntax_A03_t03.dart

```diff
@@ -2,54 +2,58 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// @assertion A rule for <viewDeclaration> is added to the grammar, along with
-/// some rules for elements used in view declarations:
+/// @assertion A rule for <inlineClassDeclaration> is added to the grammar,
+/// along with some rules for elements used in inline class declarations:
 ///
-/// <viewDeclaration> ::=
-///   'view' 'class' <typeIdentifier> <typeParameters>?
-///       <viewPrimaryConstructor>?
-///       <interfaces>?
+/// <inlineClassDeclaration> ::=
+///   'inline' 'class' <typeIdentifier> <typeParameters>? <interfaces>?
 ///   '{'
-///     (<metadata> <viewMemberDeclaration>)*
+///     (<metadata> <inlineMemberDeclaration>)*
 ///   '}'
 ///
-/// <viewPrimaryConstructor> ::=
-///   '(' <type> <identifier> ')'
-///
-/// <viewMemberDeclaration> ::=
-///   <classMemberDefinition>
+/// <inlineMemberDeclaration> ::= <classMemberDefinition>
 /// ...
-/// If a view declaration named View does not include a <viewPrimaryConstructor>
-/// then an error occurs unless the view declares exactly one instance variable
-/// v. An error occurs unless the declaration of v is final. An error occurs if
-/// the declaration of v is late.
+/// A compile-time error occurs if the inline class does not declare any
+/// instance variables, and if it declares two or more instance variables. Let
+/// id be the name of unique instance variable that it declares. The declaration
+/// of id must have the modifier final, and it can not have the modifier late;
+/// otherwise a compile-time error occurs.
 ///
-/// @description Checks that it is a compile-time error if a view declaration
-/// does not include a <viewPrimaryConstructor> and declares more than one
-/// instance variables
+/// @description Checks that it is a compile-time error if an inline class
+/// declares more than one instance variables
 /// @author sgrekhov22@gmail.com
 
 // SharedOptions=--enable-experiment=inline-class
 
-view class View1 {
+inline class IC1 {
   final int id;
   final String name = "Name";
 //^
 // [analyzer] unspecified
 // [cfe] unspecified
-  View1(this.id);
+  IC1(this.id);
 }
 
-view class View2 {
+inline class IC2 {
   int id;
+//^
+// [analyzer] unspecified
+// [cfe] unspecified
   final _name = "Name";
```

Comment:
It's hard to tell, there could be an error on `_name` because it's the second instance variable declaration: `id` could be considered to be an instance variable declaration because it is ;-), or it could be ignored because it has an error. It's the usual difficulty around derived errors, they aren't specified.

Reply:
Yes. Or may be an error on the inline class itself. Will update the tests if needed

Reply:
OK!